regression-test/suites/statistics/test_ddl.groovy

```diff
@@ -44,6 +44,7 @@ suite("test_ddl") {
     """
 
+/*
```

Comment:
add some comment to explain why disable this test, and what to do if we want to reopen it. if we do not need it anymore, remove it directly

Reply:
removed